test/example/example.pb.go

```diff
@@ -305,7 +308,8 @@ func (m *B) Unmarshal(data []byte) error {
 			if postIndex > l {
 				return io.ErrUnexpectedEOF
 			}
-			m.G = append(m.G, github_com_gogo_protobuf_test_custom.Uint128{})
```

Comment:
I regenerated the code on master now.
So if you pull in master again, this diff should go away.
My bad.